### PR TITLE
add basic password reset feature.

### PR DESCRIPTION
**Pull request by AravindaM**

### CODE REVIEW

File: lib/ui/auth/forgot_password.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              onPressed: () async {
+                await authController.resetPassword(context);
+                showDialog<String>(
+                  context: context,
+                  builder: (BuildContext context) => AlertDialog(
+                    title: const Text('Password reset email sent'),
+                    actions: <Widget>[
+                      TextButton(
+                        onPressed: () => {
+                          Navigator.pop(context, 'OK'),
```
Comment (sukie2) [inline lib/ui/auth/forgot_password.dart:45]: We can use getx back() function. https://pub.dev/packages/get#:~:text=To%20close%20snackbars%2C%20dialogs%2C%20bottomsheets%2C%20or%20anything%20you%20would%20normally%20close%20with%20Navigator.pop(context)%3B

File: lib/ui/auth/login_screen.dart [hunk 2]
```diff
@@ -46,30 +50,43 @@ class LoginScreen extends StatelessWidget {
...
-                              StringConstants.error_login_fail,
-                              snackPosition: SnackPosition.BOTTOM,
-                              duration: Duration(seconds: 3),
-                              backgroundColor:
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                    children: [
+                      TextButton(
+                        onPressed: () {
+                          Get.to(() => ForgotPasswordScreen());
```
Comment (sukie2) [inline lib/ui/auth/login_screen.dart:58]: Could use Get.to(ForgotPasswordScreen());

Comment (AravindaM) [reply lib/ui/auth/login_screen.dart:58]: Get.to(ForgotPasswordScreen()); gives a warning in the console asking to change to `Get.to(() => ForgotPasswordScreen());`

Comment (sukie2) [reply lib/ui/auth/login_screen.dart:58]: I get it :)

**Pull request merged**